diff --git a/Makefile b/Makefile
index 9283bc6..352c55e 100644
--- a/Makefile
+++ b/Makefile
@@ -1,18 +1,6 @@
-CXX_FLAGS=-O3 -std=c++11 -DNDEBUG
+CXX_FLAGS=-Ofast -std=c++11 -DNDEBUG -march=native
 INCLUDE_PATH=-I./include/
 
-ifeq ($(CXX),icpc)
-CXX_FLAGS += -qopenmp -xhost 
-else
-ifeq ($(CXX),g++)
-CXX_FLAGS += -fopenmp -march=native 
-else
-ifeq ($(CXX),clang++)
-CXX_FLAGS += -fopenmp -march=native
-endif
-endif
-endif
-
 all: lib/libhptt.so lib/libhptt.a
 
 avx: 
